packages/css/src/components/file-input/file-input.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/text-rendering";
+
+@mixin reset-button {
+  border: 0;
+}
+
+.ams-file-input {
+  align-items: center;
```

Comment:
Required to align the button text with the filename text on iOS.

Reply:
Are you sure? This doesn't seem to do anything on an iPhone 13 in Safari

Reply:
Added this rule before changing line-height and font, not needed